src/psyclone/parse/module_info.py

```diff
@@ -233,13 +246,12 @@ def get_used_symbols_from_modules(self):
         :rtype: dict[str, set[str]]
 
         '''
-        if self._used_symbols_from_module is None:
-            self._extract_import_information()
+        if self._used_symbols_from_module_name is None:
+            self._extract_used_module_names_from_fparser_tree()
```

Comment:
Hm... OK. I think it's more precise to explain what it really does. Note that the fparser information can be "wrong" because of the lack of precompiler stuff and supporting parsing non-precompiled files.

Reply:
(at least I guess this can happen)

Reply:
Anyhow, I reverted it.